### PR TITLE
feature(bank-sdk): Skonto. Amounts validation message

### PR DESCRIPTION
**Pull request by ndubkov-distcotech**
PP-763

### CODE REVIEW

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt [hunk 1]
```diff
@@ -0,0 +1,1052 @@
...
+internal fun SkontoScreenContent(
+    amountFormatter: AmountFormatter,
+    navigateBack: () -> Unit,
+    navigateToHelp: () -> Unit,
+    viewModel: SkontoFragmentViewModel,
+    modifier: Modifier = Modifier,
+    screenColorScheme: SkontoScreenColors = SkontoScreenColors.colors(),
+    isBottomNavigationBarEnabled: Boolean,
+    customBottomNavBarAdapter: InjectedViewAdapterInstance<SkontoNavigationBarBottomAdapter>?,
+    navigateToInvoiceScreen: (documentId: String, infoTextLines: List<String>) -> Unit,
```
Comment (abolfazlimahdi) [inline bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt:109]: [Suggestion] I think this should move upward (the optional parameter should be the last)

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt [hunk 2]
```diff
@@ -0,0 +1,1052 @@
...
+    amountFormatter: AmountFormatter,
+    onDiscountSectionActiveChange: (Boolean) -> Unit,
+    onSkontoAmountChange: (BigDecimal) -> Unit,
+    onFullAmountChange: (BigDecimal) -> Unit,
+    onDueDateChanged: (LocalDate) -> Unit,
+    onBackClicked: () -> Unit,
+    onHelpClicked: () -> Unit,
+    onProceedClicked: () -> Unit,
+    isBottomNavigationBarEnabled: Boolean,
+    customBottomNavBarAdapter: InjectedViewAdapterInstance<SkontoNavigationBarBottomAdapter>?,
```
Comment (abolfazlimahdi) [inline bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt:162]: [Suggestion] I think this should move down in the parameters list

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt [hunk 3]
```diff
@@ -0,0 +1,1052 @@
...
+    onHelpClicked: () -> Unit,
+    onProceedClicked: () -> Unit,
+    onInvoiceClicked: () -> Unit,
+    state: SkontoScreenState.Ready,
+    onDiscountSectionActiveChange: (Boolean) -> Unit,
+    onDiscountAmountChange: (BigDecimal) -> Unit,
+    onDueDateChanged: (LocalDate) -> Unit,
+    onFullAmountChange: (BigDecimal) -> Unit,
+    isBottomNavigationBarEnabled: Boolean,
+    customBottomNavBarAdapter: InjectedViewAdapterInstance<SkontoNavigationBarBottomAdapter>?,
```
Comment (abolfazlimahdi) [inline bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt:211]: [Suggestion] I think this should move down in the parameters list

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt [hunk 4]
```diff
@@ -0,0 +1,1052 @@
...
+        Icon(
+            painter = rememberVectorPainter(image = Icons.AutoMirrored.Default.ArrowBack),
+            contentDescription = null,
+        )
+    }
+}
+
+@Composable
+private fun InvoicePreviewSection(
+    modifier: Modifier = Modifier,
```
Comment (abolfazlimahdi) [inline bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt:403]: [Suggestion] I think this should be the last parameter in this widget.

**Review by abolfazlimahdi — CHANGES_REQUESTED** [review]
Code looks fine. Only in the SkontoScreenContent file, please take care of the parameter order (I commented in some sections, but there are more cases). 
The behavior is not as it was intended. The validation error show for a moment and then disappears. It should disappear with a user's action. Thank you!

**Comment by ndubkov-distcotech** [comment]
This PR will be replaced by [PR#619](https://github.com/gini/gini-mobile-android/pull/619)

**Pull request closed without merging**